docs-site/content/docs/the-app/models.md

```diff
@@ -126,6 +126,15 @@ When a model is added via migration, the following default fields are provided:
 
 These fields are ignored if you provide them in your migration command.
 
+### Field syntax
+
+Each field generally follows the format: `name:type[!^]`, where:
```

Comment:
I think you should also include the giver when `!` or `^` is not provided.

Reply:
You mean specify that they are optional?

Reply:
Yes we have 3 option:
1. name:string!
2. name:string^
3. name:string